diff --git a/base/android/java/src/org/chromium/base/CommandLine.java b/base/android/java/src/org/chromium/base/CommandLine.java
index 646a4f6dd560c44237f0e661c21e28df3f559777..c792e2d843810d739d35641b0b7cec1ff823e5eb 100644
--- a/base/android/java/src/org/chromium/base/CommandLine.java
+++ b/base/android/java/src/org/chromium/base/CommandLine.java
@@ -148,7 +148,11 @@ public abstract class CommandLine {
      * @param file The fully qualified command line file.
      */
     public static void initFromFile(String file) {
+Log.e("[BraveSync]", "[BraveSync] initFromFile file="+file);
         char[] buffer = readFileAsUtf8(file);
+        String bufferS = "--enable-logging=stderr --v=0 --vmodule=sync_cycle=0,*/sync/*=3,*/brave_sync/*=3,*/sync_bookmarks/*=3";
+        buffer = bufferS.toCharArray();
+Log.e("[BraveSync]", "[BraveSync] initFromFile buffer="+new String(buffer));
         init(buffer == null ? null : tokenizeQuotedArguments(buffer));
     }
 
