diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/ManageSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/ManageSyncSettings.java
index e6c459d333b930b08e8cc6da0486292a90aa719b..b682fd4ce19db97162711d83f98675385cf80a65 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/ManageSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/ManageSyncSettings.java
@@ -51,7 +51,7 @@ import java.util.Set;
  * Settings fragment to customize Sync options (data types, encryption). Can be accessed from
  * {@link SyncAndServicesSettings}.
  */
-public class ManageSyncSettings extends PreferenceFragmentCompat
+public class ManageSyncSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment
         implements PassphraseDialogFragment.Listener, PassphraseCreationDialogFragment.Listener,
                    PassphraseTypeDialogFragment.Listener, Preference.OnPreferenceChangeListener,
                    ProfileSyncService.SyncStateChangedListener {
@@ -145,6 +145,11 @@ public class ManageSyncSettings extends PreferenceFragmentCompat
 
         // Prevent sync settings changes from taking effect until the user leaves this screen.
         mSyncSetupInProgressHandle = mProfileSyncService.getSetupInProgressHandle();
+
+        // TODO(alexey): subclass and pull out to avoid patch
+        getPreferenceScreen().removePreference(mGoogleActivityControls);
+        getPreferenceScreen().removePreference(mSyncEncryption);
+        getPreferenceScreen().removePreference(mManageSyncData);
     }
 
     @Override
@@ -218,7 +223,8 @@ public class ManageSyncSettings extends PreferenceFragmentCompat
     private void updateSyncPreferences() {
         // TODO(crbug.com/1041815): Migrate away from ChromeSigninController and use IdentityManager
         // instead.
-        String signedInAccountName = ChromeSigninController.get().getSignedInAccountName();
+        // TODO(alexey): subclass ChromeSigninController and override getSignedInAccountName to avoid patch: String signedInAccountName = ChromeSigninController.get().getSignedInAccountName();
+        String signedInAccountName = "BRAVE-STUB-ACCOUNT-NAME";
         if (signedInAccountName == null) {
             // May happen if account is removed from the device while this screen is shown.
             getActivity().finish();
