diff --git a/chrome/browser/resources/settings/site_settings/category_default_setting.js b/chrome/browser/resources/settings/site_settings/category_default_setting.js
index 329851abf6153ee1d126f4424a3a7320f062f3ae..58cfcb9681c4cefcf2332d6dcfd6bee6bc3ce06f 100644
--- a/chrome/browser/resources/settings/site_settings/category_default_setting.js
+++ b/chrome/browser/resources/settings/site_settings/category_default_setting.js
@@ -185,6 +185,7 @@ Polymer({
             this.categoryEnabled ? ContentSetting.ALLOW : ContentSetting.BLOCK);
         break;
       case ContentSettingsTypes.AUTOMATIC_DOWNLOADS:
+      case ContentSettingsTypes.AUTOPLAY:
       case ContentSettingsTypes.CAMERA:
       case ContentSettingsTypes.CLIPBOARD:
       case ContentSettingsTypes.GEOLOCATION:
