diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 0606b7e68670877100114732174af25ad0b42f77..7eb8e5468a90afd938ced3c59befefe37d72e940 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -99,6 +99,8 @@ jumbo_static_library("base") {
   ]
   deps = [
     "//base:i18n",
+    "//brave/components/brave_sync:jslib_messages",
+    "//brave/components/brave_sync:pref_registration",
     "//components/os_crypt",
     "//components/pref_registry",
     "//components/prefs",
