diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 83a305858765db05548594d954a2120bd91eda58..52d998c045a699c62bc753d3fd819b83cea4c561 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -409,6 +409,7 @@ jumbo_source_set("browser_sources") {
     "//third_party/blink/public/mojom/frame",
     "//ui/display",
   ]
+  if (brave_chromium_build) { deps += [ "//brave/extensions/browser"] }
 
   public_deps = [
     "//content/public/common",
